### PR TITLE
Assessment export & import

### PR DESCRIPTION
**Pull request by Feuermagier**
Closes https://github.com/kit-sdq/intelligrade/issues/27. The export format is `<submission id>;<correction round>;<metajson string>`.

### CODE REVIEW

File: src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java [hunk 1]
```diff
@@ -200,6 +206,45 @@ public void submit() throws AnnotationMappingException, ArtemisNetworkException
         this.internalSaveOrSubmit(true);
     }
 
+    /**
+     * Exports the assessment to a string. Useful for (offline) checkpointing of
+     * assessments. The returned string contains all information necessary to
+     * recreate the exact state of the assessment, as long as the underlying
+     * submission is the same.
+     * 
+     * @return
```
Comment (dfuchss) [inline src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java:215]: Please remove incomplete parts of the JDoc :)

**Review by dfuchss — APPROVED** [review]

**Review by Luro02 — APPROVED** [review]

File: src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java (hunk 1 above)
Comment (dfuchss) [inline src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java:215]:
```suggestion
```

File: src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java [hunk 2]
```diff
@@ -200,6 +206,44 @@ public void submit() throws AnnotationMappingException, ArtemisNetworkException
...
+        return header + MetaFeedbackMapper.serializeAnnotations(this.annotations);
+    }
+
+    /**
+     * Imports the given assessment string. This overwrites the current assessment.
+     * It is checked that the submission ID and correction round match the current
+     * assessment.
+     * 
+     * @param exportedAssessment
+     * @throws AnnotationMappingException
```
Comment (dfuchss) [inline src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java:228]:
```suggestion
```

File: src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java [hunk 3]
```diff
@@ -200,6 +206,44 @@ public void submit() throws AnnotationMappingException, ArtemisNetworkException
         this.internalSaveOrSubmit(true);
     }
 
+    /**
+     * Exports the assessment to a string. Useful for (offline) checkpointing of
+     * assessments. The returned string contains all information necessary to
+     * recreate the exact state of the assessment, as long as the underlying
+     * submission is the same.
+     * 
+     * @throws AnnotationMappingException
```
Comment (dfuchss) [inline src/main/java/edu/kit/kastel/sdq/artemis4j/grading/Assessment.java:215]:
```suggestion
```

**Review by dfuchss — APPROVED** [review]

**Pull request merged**